scripts/lst-sensitivity.py

```diff
@@ -21,20 +22,28 @@
 
 args = parser.parse_args()
 
-nfiles_gammas = 100*0.2 #Pointlike gammas
-nfiles_protons = 5000*0.2
+nfiles_gammas = 100 #Pointlike gammas
+nfiles_protons = 5000*0.8
```

Comment:
These numbers (0.2 and 0.8) are coming from the division between train and test, right? what about `nfiles_gammas`?

Reply:
Yes, they come from there. Gammas are not divided because I trained with diffuse gammas and these are point like gammas, so I'm using all of them for test. 
Still I have to include another separation of files, because we need to use different events for optimizing the cuts and for calculating the sensitivity.